src/app/chat/views/communities.nim

```diff
@@ -16,6 +16,19 @@ type
     InProgress,
     Error
 
+proc ensureCorrectChatPositions(community: var Community, chat: Chat) =
+  var i = 0
+  for c in community.chats:
+    if (c.categoryId == chat.categoryId and c.id != chat.id):
+      if (c.position == 0):
+        community.chats[i].position = 1
+
+      if (c.position >= chat.position+1):
+        community.chats[i].position = c.position + 1
+
+    i = i + 1
```

Comment:
This is a WIP attempt to update the chat item `positions` **until** we get a response from status-go.

This is the part I need help with.